akka-docs/src/main/paradox/common/cluster-usage.md

```diff
@@ -0,0 +1,166 @@
+
+## Node Roles
+
+Not all nodes of a cluster need to perform the same function: there might be one sub-set which runs the web front-end,
+one which runs the data access layer and one for the number-crunching. Deployment of actors—for example by cluster-aware
+routers—can take node roles into account to achieve this distribution of responsibilities.
+
+The node roles are defined in the configuration property named `akka.cluster.roles`
+and typically defined in the start script as a system property or environment variable.
+
+The roles are part of the membership information in `MemberEvent` that you can subscribe to.
+
+## The Phi Accrual Failure Detector
```

Comment:
while we are at it and splitting up into smaller pages shall we have these sections as separate pages:
* The Phi Accrual Failure Detector
* Cluster configuration
* Cluster management

Reply:
My thought as well.